week_1/task_3_quadratic/Task_3 description.txt

```diff
@@ -0,0 +1,10 @@
+На вход вашей программе в стандартном вводе даны действительные коэффициенты A, B и C
```

Comment:
remove duplicating

Reply:
Fixed in 14541a0056ce85b5708ffbe8367445899047bc35.
Please check.